cloudmailer/cloudmailer.py

```diff
@@ -720,25 +720,57 @@ def main(argv=None):
 
     if args.schedule and not args.date:
         print ("When scheduling you need to also set a --date")
-        return 22
+        sys.exit(22)
+    # Make sure that the args date is set and have the correct format when scheduling
+    if args.schedule:
+        try:
+            datetime.strptime(args.date, '%Y-%m-%d %H:%M')
+        except ValueError:
+            print ('Starttime %s does not match the format %Y-%m-%d %H:%M')
+            sys.exit(1)
+
+    if args.schedule and args.vms:
+        print ("Cloudmailer does not currently support scheduling downtime on a VM basis. " \
+               "Only hypervisor based downtimes can be scheduled.")
+        sys.exit(1)
+
+    if args.projects and  args.schedule:
+        print ("Cloudmailer does not currently support scheduling downtime on a project basis. " \
+               "Only hypervisor based downtimes can be scheduled.")
+        sys.exit(1)
+
+    return args
+
+def get_template(template_path):
+    """Read template from file"""
+    try:
+        # One should probably define encoding here
+        with open(template_path) as template_file:
+             template = template_file.readlines()
+    except FileNotFoundError:
```

Comment:
Sames as #728, but I would not restrict the exception to file not found, but use the generic Exception to catch any other exception (permissions for example).

Reply:
I don't care enough. The most common error would be that the user define an unexisting template with `--template` . I already regret modifying this part of the code becuse of pylint since it previously looked like this: https://github.com/CSCfi/cccp-ops-contrib/pull/21/files#diff-ba58717e0e23148385741aeda5d4a5ceb3796cc7b322770fd1042e2d82a03bc7L728-L734